### PR TITLE
MGMT-19109: Add note to IBI standalone docs about machineNetwork req

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.17, 4.18

Issue: https://issues.redhat.com/browse/MGMT-19109

Link to docs preview: https://83338--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image-base-install/ibi-edge-image-based-install-standalone.html#create-standalone-config-iso_ibi-edge-image-based-install

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by trewest — APPROVED** [review]
/lgtm

**Review by achuzhoy — APPROVED** [review]

File: modules/ibi-create-standalone-config-iso.adoc [hunk 1]
```diff
@@ -72,6 +73,14 @@ pullSecret: '{"auths":{"<your_pull_secret>"}}}'
 sshKey: 'ssh-rsa <your_ssh_pub_key>'
 ----
 
+[IMPORTANT]
+====
+If your cluster deployment requires a proxy configuration, you must do the following:
+* Create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
+* Configure the machineNetwork field in your installation manifest.
```
Comment (skrthomas) [inline modules/ibi-create-standalone-config-iso.adoc:80]:
```suggestion
If your cluster deployment requires a proxy configuration, you must do the following:

* Create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
* Configure the machineNetwork field in your installation manifest.
```

Comment (skrthomas) [reply modules/ibi-create-standalone-config-iso.adoc:80]: image
You need a space before the unordered list so it renders.

**Review by skrthomas — COMMENTED** [review]
small comment, lgtm otherwise!

**Pull request merged**